src/kolibri_android/remoteshell_service/service.py

```diff
@@ -0,0 +1,125 @@
+import logging
+import os
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import serialization
+from cryptography.hazmat.primitives.asymmetric import rsa
+from kolibri.main import initialize
+from twisted.conch import manhole
+from twisted.conch import manhole_ssh
+from twisted.conch.ssh import keys
+from twisted.cred import checkers
+from twisted.cred import credentials
+from twisted.cred import error
+from twisted.cred import portal
+from twisted.internet import defer
+from twisted.internet import reactor
+from zope.interface import implementer
+
+from ..application import BaseService
+
+
+class RemoteShellService(BaseService):
```

Comment:
Do you know if this remote shell works at all? I never tried it but it would be good for debugging.

Reply:
It didn't originally even have a main function, which is _weird_. It has one now, but the service isn't being run anywhere either. I thought about doing that, but it felt out of scope :) It would definitely be great to get this up and running for real!